src/node/index.ts

```diff
@@ -1,15 +1,15 @@
 import { JSDOM } from "jsdom";
 import { Options } from "roughjs/bin/core";
+import serializer from "w3c-xmlserializer";
 
 import roughUp from "../shared";
 
 export default (input: string, options: Options = {}): string => {
-  const { window } = new JSDOM();
+  const { window } = new JSDOM(input, { contentType: "image/svg+xml" });
 
-  window.document.body.innerHTML = input;
   const svg = window.document.querySelector("svg") as SVGSVGElement;
 
   roughUp(svg, options);
 
-  return svg.outerHTML;
+  return serializer(window.document);
```

Comment:
_question_: Why `serializer(window.document)` vs `serializer(svg.outerHTML)`?

Reply:
`outerHTML` returns the elements and its descendants, but not its siblings (doctype, comments, whitespace etc).

There is a subtle difference in that previously it returned just the SVG if happened to be embedded (in HTML or whatever), now it returns the whole document.